docs/installation.md

```diff
@@ -0,0 +1,49 @@
+---
+hide:
+  - navigation
+---
```

Comment:
Why?

Reply:
tbh, I'm not sure what it does, since I'm reusing the doc from the file installation.md in getting started, it just followed through with the move

Reply:
You can use that when the page you're looking at is not in a context.
That would hide the navigation bar on the left.
It can be ok, it may not be.

Reply:
@FabienLelaquais Ah I see, in this case, the installation page is under home page, so should we also expose the nav bar on the homepage?

Reply:
Well that would make sense I guess...